files/en-us/web/api/document/querycommandstate/index.html

```diff
@@ -31,29 +31,46 @@ <h3 id="HTML">HTML</h3>
 
 <pre class="brush: html">&lt;div contenteditable="true"&gt;Select a part of this text!&lt;/div&gt;
 &lt;button onclick="makeBold();"&gt;Test the state of the 'bold' command&lt;/button&gt;
+
+&lt;hr&gt;
+
+&lt;div id="output"&gt;&lt;/div&gt;
+</pre>
+
+<pre class="brush: css hidden">
+hr {
+  margin: 1rem 0;
+}
+
+#output:before {
```

Comment:
```suggestion
#output::before {
```
Personally, I prefer to use two colons to separate pseudo elements from pseudo classes, but both ways are valid.

Reply:
`:before` is removed, so this isn't relevant any more.